PantheonCMD/pcentrypointvalidator.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/python3
+
+import argparse
+import re
+import os
+from pcchecks import Regex
+import sys
+from pcutil import get_exist, get_not_exist
+from pcprvalidator import get_no_prefix_files, get_all_modules, get_all_assemblies, get_undetermined_files
+from pcvalidator import validation
+from pcmsg import print_message, print_report_message
+
+parser = argparse.ArgumentParser()
+
+
+def get_nonexisting_entry_points(entry_point_list):
+    nonexistent_files = get_not_exist(entry_point_list)
+
+    if nonexistent_files:
+        print_message(nonexistent_files, 'entry point', 'does not exist in your repository')
+        sys.exit(2)
+
+
+def get_includes(entry_points):
+    path_to_includes = []
+
+    for entry in entry_points:
+        path_to_entry_point = os.path.dirname(os.path.abspath(entry))
+
+        with open(entry, 'r') as file:
+            original = file.read()
+            stripped = Regex.MULTI_LINE_COMMENT.sub('', original)
+            stripped = Regex.SINGLE_LINE_COMMENT.sub('', stripped)
+
+            included_files = re.findall(Regex.INCLUDED_CONTENT, stripped)
+
+            if included_files:
+
+                for include in included_files[:]:
+                    if include.startswith('_'):
+                        included_files.remove(include)
+
+                for i in included_files:
+                    path_to_includes.append(os.path.join(path_to_entry_point, i))
+
+    return path_to_includes
+
+
+def get_level_one_includes(files):
+    path_to_level_one_includes = get_includes(files)
+
+    return path_to_level_one_includes
+
+
+def get_level_two_includes(files):
+    path_to_level_two_includes = get_includes(files)
+
+    return path_to_level_two_includes
+
+
+def get_level_three_includes(files):
+    path_to_level_three_includes = get_includes(files)
+
+    return path_to_level_three_includes
+
+
+def get_level_four_includes(files):
+    path_to_level_four_includes = get_includes(files)
+
+    return path_to_level_four_includes
+
+
+def get_concatenated_includes(entry_point_list):
+    existing_entry_points = get_exist(entry_point_list)
+    level_one_includes = get_level_one_includes(existing_entry_points)
```

Comment:
These functions are acting as wrappers for a single command today.

If you'd like to keep this format, perhaps this chain can be simplified to something similar to the following -

`level_one_includes = get_includes(existing_entry_points)`
`level_two_includes = get_includes(level_one_includes)`
`...`